client/components/PerformanceMonitor.vue

```diff
@@ -51,11 +51,16 @@ const { fps, memory, renderer } = useDevtoolsHook()
         </div>
       </div>
       <div class="p4">
-        <NTip class="text-sm">
-          <Icon
-            class="mr2"
-            name="carbon:information"
-          />Memory allocated by objects is aprox <strong>{{ bytesToKB(memory.allocatedMem) }} KB </strong>
+        <NTip class="text-sm flex items-center gap-2">
```

Comment:
remove flex, items-center and gap-2 here

Reply:
Removed the css on NTip outside text-sm. Commit is pushed.